src/pkg/utils/io.go

```diff
@@ -263,11 +264,29 @@ func GetFinalExecutablePath() (string, error) {
 		return "", err
 	}
 
-	// In case the binary is symlinked somewhere else, get the final destination!!
+	// In case the binary is symlinked somewhere else, get the final destination
 	linkedPath, err := filepath.EvalSymlinks(binaryPath)
 	return linkedPath, err
 }
 
+// GetFinalExecutableCommand returns the final path to the Zarf executable including and library prefixes and overrides.
+func GetFinalExecutableCommand() (string, error) {
+	// In case the binary is symlinked somewhere else, get the final destination
+	zarfCommand, err := GetFinalExecutablePath()
+	if err != nil {
+		return zarfCommand, err
+	}
+
+	zarfCommand = fmt.Sprintf("%s %s", zarfCommand, config.ActionsCommandZarfPrefix)
```

Comment:
this will result in a double space when used within zarf, as `config.ActionsCommandZarfPrefix` will be `""`

Reply:
and `cmd = strings.ReplaceAll(cmd, "./zarf ", zarfCommand+" ")`

Reply:
I was hoping you wouldn't notice :sweat_smile: - it doesn't affect functionality, but I'll fix it lol